modules/aruco/samples/create_board_charuco.cpp

```diff
@@ -48,11 +9,11 @@ namespace {
 const char* about = "Create a ChArUco board image";
 //! [charuco_detect_board_keys]
 const char* keys  =
-        "{@outfile |<none> | Output image }"
-        "{w        |       | Number of squares in X direction }"
-        "{h        |       | Number of squares in Y direction }"
-        "{sl       |       | Square side length (in pixels) }"
-        "{ml       |       | Marker side length (in pixels) }"
+        "{@outfile |res.png| Output image }"
+        "{w        |  5    | Number of squares in X direction }"
+        "{h        |  7    | Number of squares in Y direction }"
+        "{sl       |  100  | Square side length (in pixels) }"
+        "{ml       |  60   | Marker side length (in pixels) }"
```

Comment:
The default values are set to start sample create_board_charuco.cpp without parameters